src/api/wallet.js

```diff
@@ -45,7 +45,7 @@ const walletApi = {
    */
   sendTokens(txHex, resolve) {
     const postData = {tx_hex: txHex};
-    return createRequestInstance(resolve).post('thin_wallet/send_tokens', postData).then((res) => {
+    return createRequestInstance(resolve, 300000).post('thin_wallet/send_tokens', postData).then((res) => {
```

Comment:
Maybe it should be a parameter in the config file.

Reply:
Done